log-base/src/Log/Backend/StandardOutput.hs

```diff
@@ -20,6 +21,15 @@ withSimpleStdOutLogger act = do
   logger <- stdoutLogger
   withLogger logger act
 
+-- | Create a simple jsonline logger for the duration of the given
+-- action, making sure that stdout is flushed afterwards.
+withSimpleJsonLineStdOutLogger :: (Logger -> IO r) -> IO r
+withSimpleJsonLineStdOutLogger act = do
+  logger <- mkLogger "jsonlines-stdout" $
```

Comment:
Wouldn't bulk logger be better here? I'd imagine that getting bulk messages once every second is enough as they would be sent to ES or indexed somewhere anyway as opposed to read by people who want immediate feedback (but that's what simple stdout logger is for).